javascript-timers/timeout.js

```diff
@@ -0,0 +1,5 @@
+function helloThere(greeting) {
+  document.querySelector('h1').textContent = 'Hello There';
+}
+
+setTimeout(helloThere, 600)
```

Comment:
This has the wrong delay:

---

![image](https://user-images.githubusercontent.com/16143771/123731305-bf16f900-d84c-11eb-81b4-81c8ebf66e41.png)